src/main/resources/jupyter/startup.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+##
+# This is a startup script designed to run on Leo-created Dataproc clusters.
+#
+# It starts up Jupyter and Welder processes. It also optionally deploys welder on a
+# cluster if not already installed.
+##
+
+# Templated values
```

Comment:
I wasn't a big fan of templating but GCE doesn't give a good way to pass args to these scripts. They suggest reading parameters from instance metadata but that seems unwieldy.